net/InterfaceController_admin.c

```diff
@@ -54,6 +54,9 @@ static void adminPeerStats(Dict* args, void* vcontext, String* txid, struct Allo
 
         Dict_putStringC(d, "addr", Address_toString(&stats[i].addr, alloc), alloc);
 
+        String* lladdrString = String_new(Sockaddr_print(stats[i].lladdr, alloc), alloc);
```

Comment:
For MAC addresses, `Sockaddr_print()` returns a string like:

`unknown (000008002787911ffdfb5996c4b2bfc3)` for a MAC that's '08:00:27:87:91:1f`.